pkg/compiler/check.go

```diff
@@ -34,6 +35,21 @@ func (comp *compiler) check() {
 	comp.checkDupConsts()
 }
 
+func (comp *compiler) checkComments() {
+	prefixes := []string{"include", "define", "incdir"}
```

Comment:
Nit: perhaps regexps would make it a bit shorter and decrease the nesting?

Reply:
Done